ui_framework/doc_site/src/views/form/text_area.js

```diff
@@ -0,0 +1,67 @@
+import React, {
+  Component,
+} from 'react';
+import {
+  KuiTextArea,
+} from '../../../../components';
+
+
```

Comment:
Minor nit, but can we remove double line breaks like this? I see it in a few places. A single line break would be great. :)

Reply:
No problem!